classes/XDSessionManager.php

```
@@ -248,6 +248,13 @@ public static function logoutUser($token = "")
         // authentication (via tokens) trip the first Exception as the
         // result of invoking resolveUserFromToken($token)
         session_destroy();
+
+        try {
+            $auth = new Authentication\SAML\XDSamlAuthentication();
+            $auth->logout();
+        } catch (InvalidArgumentException $ex) {
+         // This will catch when a configuration directory does not exist if it is set in the environment level
```

Shouldn't we log something here? Empty catch statements hurt my soul.

---

Nope, this is specifically when apache or nginx have been set up to to have an alternate saml config directory and that directory does not exist, so we ignore it as saml isnt set up and we dont have to do anything with it.  This is the same as what is being done in login.php

---

Lets add "this is specifically when apache or nginx have been set up to to have an alternate saml config directory and that directory does not exist, so we ignore it as saml isnt set up and we dont have to do anything with it" to the comment.